test/statscollector/DefaultStatsCollector.test.ts

```diff
@@ -21,15 +21,15 @@ import MeetingSessionStatusCode from '../../src/meetingsession/MeetingSessionSta
 import TimeoutScheduler from '../../src/scheduler/TimeoutScheduler';
 import DefaultSignalingClient from '../../src/signalingclient/DefaultSignalingClient';
 import AudioLogEvent from '../../src/statscollector/AudioLogEvent';
-import DefaultStatsCollector from '../../src/statscollector/DefaultStatsCollector';
+import StatsCollector from '../../src/statscollector/StatsCollector';
 import VideoLogEvent from '../../src/statscollector/VideoLogEvent';
 import DefaultVideoStreamIdSet from '../../src/videostreamidset/DefaultVideoStreamIdSet';
 import DefaultVideoStreamIndex from '../../src/videostreamindex/DefaultVideoStreamIndex';
 import DefaultWebSocketAdapter from '../../src/websocketadapter/DefaultWebSocketAdapter';
 import DOMMockBehavior from '../dommock/DOMMockBehavior';
 import DOMMockBuilder from '../dommock/DOMMockBuilder';
 
-describe('DefaultStatsCollector', () => {
+describe('StatsCollector', () => {
```

Comment:
Can we rename the file too?

Reply:
I renamed it yesterday but missed this change in today's PR.
Updated.